test/testenv/verificationutils.go

```diff
@@ -569,3 +569,37 @@ func VerifyAppFrameworkDeployment(standalone *enterprisev1.Standalone) {
 	}
 
 }
+
+// VerifyAppInstall verify that app of specific version is installed
+func VerifyAppInstall(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps map[string]string, versionCheck bool, statusCheck string) {
+	for appName, version := range apps {
+		status, versionInstalled, err := GetPodAppStatus(deployment, podName, ns, appName)
+		gomega.Expect(err).Should(gomega.Equal(""))
+		gomega.Expect(status).Should(gomega.Equal(statusCheck))
```

Comment:
Nit: Since its a string match can we add a comment for acceptable/expected return for `status`. Also might be a good idea to change both values to lower/upper case before comparison. We should also add log before assertion for meaningful error.